src/index.js

```diff
@@ -0,0 +1,72 @@
+import './styles/styles.css';
+
+const refresh = document.querySelector('.refresh');
+const urlsecond = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/6KPTl0XO4XB9LqSCFiZa/scores';
+const submit = document.querySelector('.submit');
+const nameList = document.querySelector('.name-lists');
+
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games';
+fetch(apiUrl, {
+  method: 'Post',
+  headers: {
+    'Content-type': 'application/json; charset=UTF-8',
+  },
+  body: JSON.stringify({
+    name: 'K42 game',
+  }),
+})
+  .then((res) => res.json())
+  .then((data) => {
+    console.log(data);
+  });
```

Comment:
- [ ] As already pointed out by the previous reviewer, please ensure to use `async and await` for your asynchronous API calls

Reply:
Hi @AnselemOdims I have done this, as you can see from my previous commit, this has been done!

So pls be more specific?

Reply:
I am very sorry, for the previous reviewer did not explain, but you can use two different ways to implement promises in javascript. We are suggesting you use` async and await` because its the better choice for readability and maintainability. 
To know how to use` async and await,` you can check [here](https://www.w3schools.com/js/js_async.asp)

so to be specific, you should replace you` then catch`  to `async await ` in your javascript code